examples/src/main/kotlin/com/teamdev/jxbrowser/examples/compose/DefaultOpenPopupCallback.kt

```diff
@@ -31,7 +31,10 @@ import com.teamdev.jxbrowser.dsl.register
 import com.teamdev.jxbrowser.dsl.removeCallback
 import com.teamdev.jxbrowser.engine.RenderingMode
 import com.teamdev.jxbrowser.view.compose.BrowserView
+import com.teamdev.jxbrowser.view.compose.popup.PopupWindow
+import com.teamdev.jxbrowser.view.compose.popup.PopupWindowState
 import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
```

Comment:
Please fix a typo below, in the comment to `val scope = rememberCoroutineScope()` field.